src/components/profile/ProfileService.js

```diff
@@ -262,11 +298,50 @@ goog.require('ga_urlutils_service');
               .attr('class', 'ga-profile-label ga-profile-label-y')
               .attr('transform', 'rotate(-90)')
               .attr('y', 0 - options.margin.left)
-              .attr('x', 0 - height / 2 - 20)
+              .attr('x', 0 - height / 2 - 30)
               .attr('dy', '1em')
+              .attr('font-size', '0.95em')
               .text($translate.instant(options.yLabel) + ' [m]');
 
-           return element;
+          //For having one pixel space below the elevation labels
+          var elevLabelHeight = height + options.margin.bottom - 1;
```

Comment:
Thi valus doesn't represent a height but the y coordinate of a position. Can you replace the name by `elevLabelY`